scripts/save_aks_credentials.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# The string to match with context.user
+MATCH_STRING="$1"
```

Comment:
MAYBE: @IonesioJunior could we select based on contexts instead of username.

Generally a kubeconfig, could contain, several cluster and several users.

Same Users could have access to multiple Cluster

but a context is unique, which is a combination of a (single cluster, single user) which is unique among the selectable variables
Thoughts?

Reply:
Ah, yes!
I didn't think about those edge cases.
I'll address that. Thank you for pointing out these details.

Reply:
Solved by: [bd4529f](https://github.com/OpenMined/PySyft/pull/8199/commits/bd4529f06a5443ad589568c769b0502b2897f617)